applications/ray/kuberay-tpu-webhook/Makefile

```diff
@@ -21,8 +25,10 @@ run: webhook
 uninstall:  
 	kubectl delete -f deployments/
 
-deploy:  
-	kubectl apply -f deployments/  
+# Create ray-system namespace (if it doesn't already exist) and deploy the webhook in-cluster
+deploy:
+	kubectl get namespace | grep -q ^${namespace} || kubectl create namespace ${namespace}
```

Comment:
Alternatively you can just specify the namespace in a manifest under `deplolyments/`
```
apiVersion: v1
kind: Namespace
metadata:
  name: ray-system
```

And `kubectl apply -f deployments/` will handle cases where the namespace already exists

Reply:
Done.